src/tools/run-make-support/src/lib.rs

```diff
@@ -102,18 +106,16 @@ fn run_common(bin_name: &str) -> (Command, Output) {
     cmd.env(&ld_lib_path_envvar, {
         let mut target_rpath_env_path = String::new();
         target_rpath_env_path.push_str(&std::env::var("TMPDIR").unwrap());
-        target_rpath_env_path.push(':');
+        target_rpath_env_path.push(path_sep(&target));
         target_rpath_env_path.push_str(&std::env::var("TARGET_RPATH_ENV").unwrap());
-        target_rpath_env_path.push(':');
+        target_rpath_env_path.push(path_sep(&target));
```

Comment:
It is the host that matters here, right? You can probably use `std::env::join_paths` here.

Reply:
Yep, that did it!